gulpfile.js

```diff
@@ -87,16 +85,28 @@ function startNodemon(done) {
   });
 }
 
+function setAvailablePort(done) {
+  const defaultPort = parseInt(process.env.PORT) || config.port;
+
+  findAvailablePort(function(port) {
+    process.env.PORT = port
+    done()
+  }, {defaultPort: defaultPort})
```

Comment:
Probably something ESLint auto fix one day

```suggestion
  }, { defaultPort })
```